src/resources/common/Configuration.ts

```diff
@@ -18,10 +18,13 @@ export enum TransferMatchAmount {
 export enum CustomerRole {
     TRANSACTION_TOKEN_READ = "transaction_token_read",
     TRANSACTION_TOKEN_UPDATE = "transaction_token_update",
-    SUBSCRIPTION_READ = "subscription_read",
+    TRANSACTION_TOKEN_CREATE = "subscription_delete",
```

Comment:
seems like values of `TRANSACTION_TOKEN_CREATE` & `SUBSCRIPTION_DELETE` got swapped?